### ISSUE TITLE
logprobs are greater than 0

### ISSUE THREAD

**Issue by RevanthRameshkumar** [marella/ctransformers#166]
for llama model .eval(), I am getting llm.logits probs that are greater than 0...which is wrong since they are supposed to be logprobs
```
00000 = {float} 1.6911218166351318
00001 = {float} -0.3545471429824829
00002 = {float} -0.2773338556289673
00003 = {float} 0.3111882507801056
00004 = {float} 2.7992610931396484
00005 = {float} 3.917383909225464
00006 = {float} 3.2658145427703857
00007 = {float} 3.7564470767974854
00008 = {float} 2.6616108417510986
00009 = {float} 1.892128586769104
00010 = {float} 2.7094860076904297
00011 = {float} 2.2569401264190674
...
```
